crates/oxc_mangler/src/lib.rs

```diff
@@ -232,13 +232,22 @@ impl Mangler {
             {
                 slots[symbol_id.index()] = assigned_slot;
 
+                let declared_scope_id =
+                    ast_nodes.get_node(symbol_table.get_declaration(*symbol_id)).scope_id();
```

Comment:
Is this getting the scope id from `symbol_id`? We can get it directly from the symbol id I think.

Reply:
It seems that does not work.
What I want is the scope id of the declaration node.
```js
function foo() { // let's say this function has scope_id = 1
  { // let's say this block has scope_id = 2
    var a;
  }
}
```
`symbol_table.get_scope_id(*symbol_id)` returns `1`, but I want `2` here.